fixtures/small/method_chains_expected.rb

```diff
@@ -15,6 +15,26 @@
     b: ""
   )
 
+foo.bar.baz
+
+foo
+  .bar
+  .baz
+
+# If they're all on the same line but different from
+# the first receiver, consider that "on one line"
+foo.bar.baz
+
+foo::bar&.nil?
+
+foo::bar
+  &.nil?::klass
+  .true?
```

Comment:
This is `blursed` but technically legal -- I handled it by just multilining _where possible_, but open to other thoughts on how we should handle `::` calls.

Reply:
I hate you

Reply:
(j/k obvi)

Reply:
I think pretty much every fixture I write should just have `# im so sorry` at the top of them